src/fs.rs

```diff
@@ -109,6 +109,10 @@ impl<Storage: driver::Storage> Allocation<Storage> {
         assert!(attr_max <= 1_022);
         // limitation of ll-bindings
         assert!(attr_max == 1_022);
+        // limitation of ll-bindings
+        // metadata_max defaults to block_size when zero
+        let metadata_max = 0;
+        assert!(metadata_max <= block_size);
```

Comment:
this seems like a no-op? (because `metadata_max` is always zero?)

Reply:
I removed the assert.   The idea was I wanted a placeholder in case this becomes a configurable value and it becomes important to know the metadata block size needs to be < block_size (a limitation imposed by the littlefs project - something that wasn't clear at this layer).